tools/generate_events.py

```diff
@@ -34,8 +34,17 @@
 type {{eventName}} struct {
     eventName string
 {%- for p, t in event.properties.items() %}
-    {{event.pascal_case(p)}} {{event.go_type(t)}}
+    {%- if event.type != "host" %}
+        {{event.pascal_case(p)}} {{event.go_type(t)}}
```

Comment:
Even though the indentation here makes this more readable, I think this is what's causing the indentation issues in the generated file. Probably just remove the indentation from the part that's going to render the go source.